docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+services:
+  isaac-sim:
+    image: vip/nvsynth
+    build:
+      context: .
+      dockerfile: docker/Dockerfile
+      target: isaac-sim
+    volumes:
+      - .:/nvsynth
+      # food data volume
+      - /pub2/nrc/aging/snair_iccv_data:/food
+      # isaac-sim related volumes
+      - ~/docker/isaac-sim/cache/kit:/isaac-sim/kit/cache/kit
+      - ~/docker/isaac-sim/cache/ov:/root/.cache/ov
+      - ~/docker/isaac-sim/cache/pip:/root/.cache/pip
+      - ~/docker/isaac-sim/cache/glcache:/root/.cache/nvidia/GLCache
+      - ~/docker/isaac-sim/cache/computecache:/root/.nv/ComputeCache
+      - ~/docker/isaac-sim/logs:/root/.nvidia-omniverse/logs
+      - ~/docker/isaac-sim/data:/root/.local/share/ov/data
+      - ~/docker/isaac-sim/documents:/root/Documents
+    environment:
+      - ACCEPT_EULA=Y
+      - PRIVACY_CONSENT=Y
+    network_mode: "host"
+    ipc: host
+    tty: True
+    runtime: nvidia
+    deploy:
+      resources:
+        reservations:
+          devices:
+            - capabilities: [gpu]
```

Comment:
Do you know if 2023 lets you use any GPU or do we still have to stick with the GPU connected to the display? If not, would be nice to have a way to specify the gpu ID the script should run on. something like this should work

```yaml
          devices:
            - driver: nvidia
              # Change to ['0', '1'] to use two GPUs or specify `GPU_IDS=0,1` before running `docker compose run`
              device_ids: ["${GPU_IDS:-0}"]
              capabilities: [gpu]
```

Reply:
Ah nvm you're right, the container is exposed to all the GPUs and the script will limit visibility to specific ones.